### DISCUSSION TITLE
Reapply application keeps failing.

### DISCUSSION THREAD

**Discussion by saivishnu725** [community/community#60773] [GitHub Education] [answered]
### Hi everyone,

I have my annual fee receipt as proof of my college admission. It is a receipt from the bank that I paid the fee from. Using this as the proof worked last year but reapplying with the same proof failed 5 times. The problem is that it has `3 sem` as my current semester and I am in `4 sem`. It also is dated `30/09/2022`.
I didn't receive any email from my college, hence I used my primary personal email account and I am also not able to find my college in the affiliated colleges' list.

[image of the date and semester](https://www.dropbox.com/s/uzs1fpcp8f56cpw/receipt.jpg)

this is the email that I received. 
```
Hey Sai Vishnu M,

Unfortunately, we were unable to verify your academic affiliation based on the information you submitted on Jul 13, 2023 for [@saivishnu725](https://github.com/saivishnu725).

Common verification problems and suggested solutions are documented here: https://docs.github.com/en/education/explore-the-benefits-of-teaching-and-learning-with-github-education/github-global-campus-for-students/why-wasnt-my-application-to-global-campus-for-students-approved.


Once you have resolved the issues with your sumbission, [visit us again and reapply](https://education.github.com/).

Have questions? Join the [GitHub Education Community](https://github.com/orgs/community/discussions/categories/github-education) for answers.

💖 The GitHub Education Team
```

**Comment by saivishnu725** [comment, marked as answer]
Update: I tried to use a screenshot of my college portal as proof and it got approved.

Found this here, https://education.github.com/discount_requests/application
```
Approved Submitted Jul 13, 2023
Congrats! Your request for [@saivishnu725](https://github.com/saivishnu725) was approved and your benefits will be available soon.
```

**Reply by JavedTadavi** [reply]
Hi Brother Same problem how to fix it I try to reach GitHub contact support but no reply

**Comment by amanmunjekar** [comment]
Subject: Request to Reinstate Access to GitHub Student Developer Pack

Dear GitHub Support,

. My name is Aman Munjekar, and I am writing to request the reinstatement of my access to the GitHub Student Developer Pack.

I am a current user of GitHub and have previously been approved for the GitHub Student Developer Pack [6/5/22 of myr previous approval]. However, due to an accidental resubmission of my application, my access to the student developer pack has been revoked.

I apologize for any inconvenience caused by this mistake.

I would greatly appreciate it if you could reinstate my access to the GitHub Student Developer Pack.

Please let me know if there are any additional steps I need to take or any further information you require to resolve this matter. I am more than willing to provide any necessary details.

Thank you for your attention to this matter. I look forward to your prompt response and the opportunity to regain access to the GitHub Student Developer Pack.

Best regards,

Aman Munjekar.
username: amanmunjekar.

**Reply by samus-aran** [reply]
Hi @amanmunjekar  👋

We’re happy you’re here but let’s not divert this discussion as this conversation is off-topic. Please open up a separate discussion so that we can look at your query there. 👍

> I am a current user of GitHub and have previously been approved for the GitHub Student Developer Pack [6/5/22 of myr previous approval].

This is actually expected as verification can happen between 1-2 years, or else we detect suspicious and fraudulent activity.

**Comment by amanmunjekar** [comment]
Hi sir,

**Comment by amanmunjekar** [comment]
Use admission card

**Comment by ghost** [comment]
HELLO GUYS IAM NEW AND I LIKE TO GET STUDENT BENEFIT FOR CANVA FREE TRAIL AND I WANT TO LOG IN BUT ITS TELL TO UPLOAD THE SCHOOL ID BUT I UPLOAD BUT ITS NOT WORKING.
![Screenshot (76)](https://github.com/community/community/assets/141650422/95f32218-fefb-48c8-9980-e59030857789)
